protobuf/types/service.proto

```diff
@@ -54,7 +54,48 @@ message Service {
     string command = 5;               // Command to run the container.
     repeated string env = 9;          // Default env vars to apply to dependency on runtime.
   }
-  
+
+  // A workflow is a configuration to trigger a specific task when certains conditions of a trigger are valid.
+  message Workflow {
+
+    // Trigger is the configuration that will trigger a workflow.
+    message Trigger {
+
+      // Type of trigger available. Event to react to an event. Result to react to a result of an execution.
+      enum TriggerType {
+        Event = 0;  // React to event.
```

Comment:
Here types are enumerated from 0, but in go code from 1.

Reply:
proto doesn't allow enum to start with 0
In Go, it starts with 1 to make sure that we have a value and then be able to do validation without having to manage empty values https://github.com/mesg-foundation/engine/pull/1188#discussion_r304224737

Reply:
Let's add a new value `UNKNOWN = 0` to be consistent with the go struct